src/test/java/com/purbon/kafka/topology/PrincipalManagerTest.java

```diff
@@ -87,19 +89,19 @@ public void testFreshGeneration() throws IOException {
     Topology topology = new TopologyImpl();
     topology.setContext("context");
     Project project = new ProjectImpl("foo");
-    project.setConsumers(Collections.singletonList(new Consumer("consumer")));
-    project.setProducers(Collections.singletonList(new Producer("producer")));
+    project.setConsumers(Collections.singletonList(new Consumer("consumer-principal")));
```

Comment:
Why is the "-principal" required?

Reply:
Adding '-principal' was just for clarification purposes - at first I was confused myself while looking at the test-code, not realizing that "consumer" was the name of the principal in this snippet `new Consumer("consumer")` 

I can revert the '-principal' adjustment, no problem..

Reply:
no, no ... if it has helped you, probably is very much required 😄 ! I was only curious to know why... jejejej